sound/soc/sof/ops.c

```diff
@@ -142,37 +142,40 @@ void snd_sof_dsp_update_bits_forced(struct snd_sof_dev *sdev, u32 bar,
 EXPORT_SYMBOL(snd_sof_dsp_update_bits_forced);
 
 int snd_sof_dsp_register_poll(struct snd_sof_dev *sdev, u32 bar, u32 offset,
-			      u32 mask, u32 target, u32 timeout)
+			      u32 mask, u32 target, u32 time)
```

Comment:
it'd be nicer if this was timeout_ms. we will get this feedback from Andy so let's anticipate :-)

Reply:
OK, let me change it.